frontend/stylesheets/_overrides.scss

```diff
@@ -379,3 +335,66 @@ footer nav li a:hover {
 }
 
 /*-----  End of Temporary Fixes  ------*/
+
+.page-get-started {
+  font-size:18px;
+}
+/*======================================
+=            Fancy Terminal            =
+======================================*/
+
+.fancy-terminal {
+  margin:20px 0;
+  padding:30px 10px 10px;
+  background: #333;
+  font-family:$font-family-monospace;
+  border-radius:6px;
+  background-image: linear-gradient(#ddd 0, #aaa 20px,#333 20px);
+}
+
+.fancy-terminal p {
+  color:lighten($hootcode-brand, 20%);
+  font-weight:bold;
+  margin:10px;
+}
+.fancy-terminal p:before {
+  content:"~ ";
+  color:cyan;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;  
+}
+
+/*-----  End of Fancy Terminal  ------*/
+
+.instruction-cards {
+  font-size:18px;
+}
+.instruction-cards .panel {
+  margin-bottom:4em;
+  position: relative;
+}
+.instruction-cards .panel-body {
+  padding:4em;
+}
+.panel-step {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  background-color: rgb(206, 107, 5);
+  left: -25px;
+  border-radius: 50%;
+  top: -25px;
+  box-shadow: 2px 2px 4px 0 rgba(0,0,0,.3);
+  text-align: center;
+  line-height: 50px;
+  font-size: 32px;
+  color: #fff;
+}
+.hootcode-banner {
+  background:url(http://localhost:4567/img/bg_header_big.png);
```

Comment:
Colon after property should be followed by one space<br>URL `http://localhost:4567/img/bg_header_big.png` should not contain protocol or domain<br>URLs should be enclosed in quotes